code42.json

```diff
@@ -3020,16 +3066,125 @@
                     "description": "Names of all the open browser tabs or windows during exposure.",
                     "data_type": "string",
                     "order": 13,
-                    "primary": false
+                    "primary": false,
+                    "contains": [
```

Comment:
What is the benefit of adding a custom contains type to parameters like `window title` that are only used on a single action? I could see a use case for this contains if it were an input or output to a different action so you could chain them together, but is it much use as is?

Reply:
We can make searches using window title, so we can output one search into another

Reply:
Oh, I see that param is not included in the output for some reason! No worries, I can add it.
I think all the search params for file events should include the data in the output so we can support doing this chain of searches

Reply:
Check it out!  Support for window title chained queries is now a thing:
<img width="567" alt="Screen Shot 2021-06-22 at 08 00 15" src="https://user-images.githubusercontent.com/19540978/122928996-03c40100-d330-11eb-9444-b3d64bbe2e39.png">

Reply:
Oooh nice this is very cool.

Reply:
Oooh nice this is very cool!